components/kubearchive/development/vacuum.yaml

```diff
@@ -0,0 +1,34 @@
+---
+apiVersion: kubearchive.org/v1
+kind: ClusterVacuumConfig
+metadata:
+  name: vacuum-config-all
+spec:
+  namespaces: {}
+---
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: vacuum-all
+spec:
+  schedule: "* */1 * * *"
```

Comment:
Should we start with a less aggressive schedule rather than hourly?  We're not sure of the impact on the system when a vacuum runs.  Maybe let's start with once or twice a day and see how it goes?

Reply:
I changed it to run once a day, close to midnight but not exactly to avoid overlapping other stuff.

Reply:
Shouldn't this be `* */24 * * *"?  Won't what you have still run once an hour?

Reply:
"20 1 * * *" makes it run at 1:20 AM, so once a day.

Reply:
Sorry, I didn't see the changes.